warrior/Framework/Utils/cli_Utils.py

```diff
@@ -174,125 +113,88 @@ def connect_telnet(ip, port="23", username="", password="",
+    elif isinstance(child, pexpect.spawn):
+        wc_obj = WNetwork.warrior_cli_class.WarriorCli()
+        wc_obj.conn_obj = WNetwork.warrior_cli_class.PexpectConnect()
+        wc_obj.conn_obj.target_host = child
+        wc_obj.disconnect_telnet()
+
     return child
 
 
 def disconnect(child):
     """
-    - Disconnects a pexpect session
+    - Disconnects warrior_cli_class session object(pexpect/paramiko)
     - Returns session object(same child)
     """
     print_warning("This method is obsolete and will be deprecated soon. Please"
                   " use 'disconnect' method of 'PexpectConnect' class "
-                  "in 'warrior/Framework/ClassUtils/warrior_connect_class.py'")
+                  "in 'warrior/Framework/ClassUtils/WNetwork/warrior_cli_class.py'")
+
+    if isinstance(child, WNetwork.warrior_cli_class.WarriorCli):
+        child = child.disconnect()
+    elif isinstance(child, pexpect.spawn):
+        wc_obj = WNetwork.warrior_cli_class.WarriorCli()
+        wc_obj.conn_obj = WNetwork.warrior_cli_class.PexpectConnect()
+        wc_obj.conn_obj.target_host = child
+        wc_obj.disconnect()
 
-    if child.isalive():
-        if child.ignore_sighup:
-            child.ignore_sighup = False
-        child.close()
     return child
 
-@cmdprinter
-def send_command_and_get_response(sessionobj, prompt1, prompt2, command):
+
+def send_command_and_get_response(session_object, start_prompt, end_prompt, command):
     """"Sends a command to a terminal expects a completion prompt
     If completion prompt was found, returns the response of the command """
-    response = ""
-    try:
-        boolprompt = sessionobj.expect(prompt1)
-    except Exception as exception:
-        print_info("Could not find the prompt "+prompt1)
-        print_exception(exception)
-
-    if boolprompt == 0:
-        sessionobj.sendline(command)
-        try:
-            sessionobj.expect(prompt2)
-        except Exception as exception:
-            print_info("Could not find the prompt "+prompt2)
-            print_exception(exception)
-        else:
-            response = sessionobj.before
+
+    print_warning("This method is obsolete and will be deprecated soon. Please"
+                  " use 'send_command' method of 'PexpectConnect' class "
+                  "in 'warrior/Framework/ClassUtils/WNetwork/warrior_cli_class.py'")
+
+    if isinstance(session_object, WNetwork.warrior_cli_class.WarriorCli):
+        _, response = session_object.send_command(start_prompt, end_prompt, command)
+    elif isinstance(session_object, pexpect.spawn):
+        wc_obj = WNetwork.warrior_cli_class.WarriorCli()
+        wc_obj.conn_obj = WNetwork.warrior_cli_class.PexpectConnect()
+        wc_obj.conn_obj.target_host = session_object
+        _, response = wc_obj.send_command(start_prompt, end_prompt, command)
+    else:
+        response = ""
+        print_warning("Unable to send the command since the session_object is not an "
```

Comment:
change print statement as '....session object is not supported by warrior_cli_class...' instead of 'session_object is not an  instance of warrior_cli_class..'